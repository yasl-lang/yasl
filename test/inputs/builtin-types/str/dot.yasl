##special_name\n10\n

echo .special_name

const x = {}

x.for = 10

echo x.for
