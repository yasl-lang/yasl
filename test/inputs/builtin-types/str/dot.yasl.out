special_name
10
